src/main/resources/data/minecraft/tags/items/music_discs.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
I'd have to get cyber's opinion on this one but I think it's good to leave for now, I personally would like users to have a random chance of finding this thing since right now the only way to get it is crafting, and we hide it in JEI :3